.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Report errors and problems
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+**Symfony version(s) affected**
+x.y.z
```

Comment:
Please note that markdown doesn't allow for new lines inside paragraphs. This (and all below) will be rendered as `**Symfony verion(s) affect** x.y.z` (see also the visual diff of this PR).

Are you aware of this effect? There are a couple solutions:

* explicitly add `<br>`
* add two spaces at the end of the first line
* use a special headline markdown (e.g. `### Symfony verion(s) affected`)

Reply:
Or add a `:` at the end of the first line.
New lines in the code help identifying the placeholders to replace and IMHO "**Symfony version(s) affected**: 3.4" inlined is as much good as rendering a new line (same for other paragraphs).

But otherwise, let's add the required two spaces for new line. 👍